ui/tests/dashboard.spec.ts

```diff
@@ -57,45 +69,45 @@ describe('dashboard', function () {
         const apiKey = process.env.BITTREX_API_KEY as string;
         const apiSecret = process.env.BITTREX_API_SECRET as string;
 
-        before(async () => {
-            await retry( async() => {
-                navigateTo(client,  '#user_settings_button');
-            });
-            await closeAddYourSettingsPopup(client);
+        before(async function () {
+            await setupTest(app, `${title}_after_adding`, async () => {
+                await navigateTo(client, '#user_settings_button');
+                await closeAddYourSettingsPopup(client);
 
-            await client.waitForVisible('#blockchain_balances_panel_body', METHOD_TIMEOUT);
+                await client.waitForVisible('#blockchain_balances_panel_body', METHOD_TIMEOUT);
 
-            await controller.addAccount(AccountType.ETH, ethAccount);
-            await controller.addAccount(AccountType.BTC, btcAccount);
+                await controller.addAccount(AccountType.ETH, ethAccount);
+                await controller.addAccount(AccountType.BTC, btcAccount);
 
-            await client.waitForExist('#btcchain_per_account_table', METHOD_TIMEOUT);
+                await client.waitForExist('#btcchain_per_account_table', METHOD_TIMEOUT);
 
-            await controller.addFiatValue();
-            await controller.addExchange(apiKey, apiSecret);
+                await controller.addFiatValue();
+                await controller.addExchange(apiKey, apiSecret);
 
-            await navigateToDashboard(client);
+                await navigateToDashboard(client);
 
-            await logout(client);
-            await login(client, username, password);
-        });
+                await logout(client);
+                await login(client, username, password);
+            });
 
-        it('should show the information to the user', async () => {
-            await client.waitForExist('#blockchain_box', METHOD_TIMEOUT);
-            await client.waitForExist('#banks_box', METHOD_TIMEOUT);
-            await client.waitForExist('#bittrex_box', METHOD_TIMEOUT);
-            await client.waitForExist('#table_balances_total_body > tr:nth-child(3)');
+            it('should show the information to the user', async () => {
```

Comment:
I have to check these whitespace changes though because the diff is confusing

Reply:
It seems that this part was broken during the manual merge that happened after I rebased on master